api/controllers/EventController.ts

```diff
@@ -81,6 +81,13 @@ export class EventController {
     return { error: null };
   }
 
+  /**
+   * Gets the event using a uuid
+   *
+   * @param params the uuid of the event to look up
+   * @param user the user who called the route
+   * @returns a response containing the event
+   */
```

Comment:
nit: I don't think we need these TS docs, since the route itself with its naming and parameters is fairly straightforward in its intention. Also, more strongly, if we add TS docs in 1 route we should try to be consistent by having them added in all routes, which is an unnecessary hassle when generally a combination of reading the method name and its parameter names is enough to convey to a developer the purpose of that route